src/Tribe/Metabox.php

```diff
@@ -217,7 +217,7 @@ public function ajax_ticket_add() {
 			 */
 			do_action( 'tribe_tickets_ticket_added', $post_id );
 		} else {
-			wp_send_json_error( esc_html__( 'Failed to Add the Ticket', 'event-tickets' ) );
+			wp_send_json_error( sprintf( esc_html__( 'Failed to add the %s', 'event-tickets' ) ), tribe_get_ticket_label_singular( 'ajax_ticket_add_error' ) );
```

Comment:
This only escapes the __ string, not the strings you are inserting via printf(). Can you follow the other examples where I've made suggestions that does it like `esc_html( sprintf( .... ) )` here elsewhere in your PR?